Source/WebKit/WPEPlatform/wpe/wayland/WPEDisplayWayland.cpp

```diff
@@ -219,32 +220,34 @@ const struct wl_registry_listener registryListener = {
         auto* display = WPE_DISPLAY_WAYLAND(data);
         auto* priv = display->priv;
 
-        if (!std::strcmp(interface, "wl_compositor"))
+        auto interfaceSpan = unsafeSpan(interface);
+
+        if (!equalSpans(interfaceSpan, "wl_compositor"_span))
```

Comment:
I guess we need to invert the logic, I assume `equalSpans`, that returns a bool, returns true if spans are equal and false otherwise.

Reply:
oooops. And I should had seen it, I was following another PR where they had done that. Thanks, Carlos, sorry